packages/tupaia-web/src/pages/Base.tsx

```diff
@@ -0,0 +1,20 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import { useLocation } from 'react-router-dom';
+import { LandingPage, Project } from '.';
+
+export const Base = () => {
+  const { pathname } = useLocation();
+
+  // This is to handle trailing '/' in a pathname
+  const finalURLChar = pathname[pathname.length - 1];
+  const finalCharSubtractor = finalURLChar === '/' ? 1 : 0;
+
+  // If a pathname has only 1 segment, treat as a landing page, otherwise treat as a project
+  const isLandingPage = pathname.split('/').length - 1 - finalCharSubtractor === 1;
```

Comment:
```suggestion
  const { entityCode } = useParams();

  const isLandingPage = !entityCode;
```
What about just using the named param to check if it's a landing page?

Reply:
Actually I think we can do away with Base altogether and just have this check in the Router